Gemfile

```diff
@@ -50,7 +50,7 @@ gem 'decision_trees', '~> 2.0.0'
 gem 'feedback', '~> 0.3.0'
 gem 'mortgage_calculator', '~> 1.3.5'
 gem 'pensions_calculator', '~> 0.2'
-gem 'rio', git: 'git@github.com:moneyadviceservice/rio.git', ref: 'bb267ca11932fd7958077d89f9802c9ae2d60d5b'
+gem 'rio', git: 'git@github.com:moneyadviceservice/rio.git', ref: '35fc40dd3adbd31124661ea66278fdad3bd6c23b'
```

Comment:
please commit updated `Gemfile.lock`, as it still points to `bb267ca11932fd7958077d89f9802c9ae2d60d5b` revision